core/claims/claim.go

```diff
@@ -110,11 +115,8 @@ var (
 	ClaimTypeAuthEthKey = NewClaimTypeNum(9)
 )
 
-// ClaimVersionLen is the length in bytes of the version in a claim.
-const ClaimVersionLen = 32 / 8
-
 // ClaimTypeVersionLen is the length in bytes of the version and length in a claim.
-const ClaimTypeVersionLen = ClaimTypeLen + ClaimVersionLen
+const ClaimTypeVersionLen = ClaimTypeLen + ClaimFlagsLen + ClaimVersionLen
```

Comment:
Technically this should be now ClaimTypeFlagsVersionLen

Reply:
Proposal: Define a constant size header of the Claim that contains:
type || flags || expiration? || version?

```
const ClaimHeaderLen = ClaimTypeLen + ClaimFlagsLen + ClaimExpirationLen + ClaimVersionLen
```

Reply:
Yes, this was still here because is used by the old implementation of the old claims that are not updated yet. But in the new implementation this is not used anymore

Reply:
Let's do it in a future commit then :)